programs/odbc-bridge/CMakeLists.txt

```diff
@@ -30,12 +30,7 @@ target_link_libraries(clickhouse-odbc-bridge PRIVATE
 set_target_properties(clickhouse-odbc-bridge PROPERTIES RUNTIME_OUTPUT_DIRECTORY ..)
 target_compile_options (clickhouse-odbc-bridge PRIVATE -Wno-reserved-id-macro -Wno-keyword-macro)
 
-if (SPLIT_DEBUG_SYMBOLS)
```

Comment:
Do I understand correctly that we won't ship debug info for the two libs?

I assume (but not sure about it) we build `RelWithDebInfo` packages, then split the debug symbols from the binaries and ship them in separate package. With this change, we will ship the debug symbols with `clickhouse-odbc-bridge` and `clickhouse-library-bridge`, right?

Reply:
Yes, all true.